assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -230,6 +231,25 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: false,
 	},
+	'auth-error': {
+		Component: AuthError,
+		priority: 130,
+		areaSlug: NOTIFICATION_AREAS.ERRORS,
+		viewContexts: [
+			VIEW_CONTEXT_MAIN_DASHBOARD,
+			VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+			VIEW_CONTEXT_ENTITY_DASHBOARD,
+			VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY,
```

Comment:
Is it possible for an auth error to exist in a view-only context? I would think not since the user is inherently non-authenticated.

Reply:
Thanks @aaemnnosttv , good shout. I have remove the view only dashboard and context constants here.